src/storage/src/table/mod.rs

```diff
@@ -108,24 +108,38 @@ pub trait TableIter: Send {
     }
 }
 
-/// Get vnode value with `indices` on the given `row`. Should not be used directly.
+/// Get vnode value with `indices` on the given `row`.
 fn compute_vnode(row: &Row, indices: &[usize], vnodes: &Bitmap) -> VirtualNode {
     let vnode = if indices.is_empty() {
         DEFAULT_VNODE
     } else {
-        row.hash_by_indices(indices, &CRC32FastBuilder {})
+        row.hash_by_indices(indices, &Crc32FastBuilder {})
             .to_vnode()
     };
 
     tracing::trace!(target: "events::storage::storage_table", "compute vnode: {:?} key {:?} => {}", row, indices, vnode);
 
-    // FIXME: temporary workaround for local agg, may not needed after we have a vnode builder
```

Comment:
As we already introduced `vnode_index` in #5079, this is unnecessary now.

Reply:
Oops. Looks like not all local agg goes to `vnode_in_col` branch, so this makes e2e fail. 🥵

Reply:
I've opened an issue at #5420.